src/actions/index.js

```diff
@@ -50,16 +50,41 @@ export const login = values => (dispatch) => {
 
 export const forgotPassword = (values, callback) => (dispatch) => {
   dispatch({ type: types.FORGOT_PASS_ATTEMPT });
-  // firebase
-  //   .auth()
-  //   .sendPasswordResetEmail(values.email)
-  //   .then((userCredential) => {
-  //     dispatch({ type: types.FORGOT_PASS_GUCCI, userCredential });
-  //     callback();
-  //   })
-  //   .catch((error) => {
-  //     dispatch({ type: types.FORGOT_PASS_FAIL, error });
-  //   });
+  axios
+    .post(`${API_PATH}/forgot-password/request`, {
+      email: values.email,
+    })
+    .then(() => {
+      dispatch({ type: types.FORGOT_PASS_GUCCI });
```

Comment:
lmao forgot_pass_gucci

Reply:
ya @mianuddin did this, ngl pretty salty